http/src/main/resources/reference.conf

```diff
@@ -34,7 +34,7 @@ dataproc {
   }
   # This image is used for legacy jupyter image where hail is compatible with earlier version of spark
   legacyCustomDataprocImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-dataproc-1-2-79-debian9-18fcafd"
-  customDataprocImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-dataproc-1-4-51-debian10-2021-01-11"
+  customDataprocImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-dataproc-1-4-51-debian10-2021-01-15"
```

Comment:
@rtitle  this is the only reference I'm aware...but lmk if there's anything else I should update

Reply:
that should be it